gef.py

```diff
@@ -10647,6 +10686,26 @@ def invoke(self, args, from_tty):
         gdb.execute("gef help")
         return
 
+    def add_context_pane(self, pane_name, pane_function):
+        """Add a new context pane to ContextCommand"""
+        context_obj = None  # type: ContextCommand
```

Comment:
This definition is unncessary in Python (yeah, it's weird that way) but this work perfectly:

```python
>>> for i in range(10):
...  if i == 2:
...   a = i
...
>>> print(a)
2
```

Which we can use because `context` will always (or should always) exist.